typing/typecore.ml

```diff
@@ -2596,6 +2596,238 @@ let rec final_subexpression exp =
     -> final_subexpression e
   | _ -> exp
 
+let is_prim ~name funct =
```

Comment:
For reviewers: this function and `list_labels{,_aux}` below are just moved upwards from where they were in the file.